bookkeeper-server/src/test/java/org/apache/bookkeeper/tls/TestTLS.java

```diff
@@ -99,6 +101,10 @@ public TestTLS(String keyStoreFormat, String trustStoreFormat) {
         this.serverTrustStoreFormat = TLSContextFactory.KeyStoreType.valueOf(trustStoreFormat);
     }
 
+    private String getResourcePath(String resource) throws Exception {
+        return URLDecoder.decode(this.getClass().getClassLoader().getResource(resource).getPath(), UTF_8.name());
```

Comment:
I think it is probably better to use `toURI().getPath()`, rather than us doing the magic decoding. 

[URI.getPath](https://docs.oracle.com/javase/7/docs/api/java/net/URI.html#getPath())